.github/workflows/webpage.yml

```diff
@@ -36,11 +42,22 @@ jobs:
         git fetch origin gh-pages
         git checkout origin/gh-pages plugins_metadata.json || true
         mv plugins_metadata.json cloned_plugins_metadata.json || true
+
+    - name: Restore cached Plugins
+      id: cache-plugins-restore
+      uses: actions/cache/restore@v3
+      with:
+        path: |
+          plugins_metadata.json
+        key: plugins_metadata
+
```

Comment:
This could potentially cause the problem that webpage will not update in daily build.